core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -661,6 +661,16 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.SERVER)
           .build();
+  public static final PropertyKey UNDERFS_OBJECT_BREADCRUMBS_ENABLED =
+      new Builder(Name.UNDERFS_OBJECT_BREADCRUMBS_ENABLED)
+          .setAlias("alluxio.underfs.object.breadcrumbs.enabled")
```

Comment:
@ns1123 is there a reason that this alias is set? It looks the same as `Name.UNDERFS_OBJECT_BREADCRUMBS_ENABLED`

Reply:
No reason. That shouldn't be there.